content/authors/Taves-mwangi/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Taves Mwangi
+type: authors
+github : https://github.com/kmwangi-ken
+---

```

Comment:
You seem to be missing something.

Please have a look at the documentation or refer files of other authors to correct this.